mls-rs-crypto-rustcrypto/src/ecdh.rs

```diff
@@ -97,6 +97,10 @@ impl DhType for Ecdh {
     fn secret_key_size(&self) -> usize {
         self.0.secret_key_size()
     }
+
+    fn public_key_size(&self) -> usize {
+        self.0.secret_key_size()
```

Comment:
Should the call be forwarded to `public_key_size` instead?
```suggestion
        self.0.public_key_size()
```